lms/djangoapps/certificates/admin.py

```diff
@@ -46,8 +47,16 @@ class CertificateTemplateAssetAdmin(admin.ModelAdmin):
     list_display = ('description', '__unicode__')
 
 
+class GeneratedCertificateAdmin(admin.ModelAdmin):
+    """
+    Django admin customizations for GeneratedCertificae model
```

Comment:
And while we're here, can we fix the typo at [here](https://github.com/edx/edx-platform/blob/rc/2015-11-03/common/djangoapps/student/views.py#L1036-L1038)? `certicifate` -> `certificate`.